test/metabase/api/user_test.clj

```diff
@@ -377,14 +377,6 @@
 (deftest create-user-validate-input-test
   (testing "POST /api/user"
     (testing "Test input validations"
-      (is (= {:errors {:first_name "value must be a non-blank string."}}
-             (mt/user-http-request :crowberto :post 400 "user"
-                                   {})))
-
-      (is (= {:errors {:last_name "value must be a non-blank string."}}
-             (mt/user-http-request :crowberto :post 400 "user"
-                                   {:first_name "whatever"})))
-
```

Comment:
Since we're not testing every optional key in this test, I followed suit and took out the (now) optional name keys tests.